kiwix-desktop.pro

```diff
@@ -18,7 +18,7 @@ greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
 TARGET = kiwix-desktop
 TEMPLATE = app
 
-QMAKE_CXXFLAGS += -std=c++17 -Werror
+QMAKE_CXXFLAGS += -std=c++17
```

Comment:
Can't we disable `-Werror` only under Windows?

Reply:
We have a general problem that Windows compiler options are not the same as gcc. We have a similar situation with the cpp17 option.

Reply:
@kelson42 it is not clear to me what the issue is. Can we not do something like the below?
```
!win32 {
    QMAKE_CXXFLAGS += -Werror
}
```

Reply:
Done.

Successful build with last change is https://ci.appveyor.com/project/Kiwix/kiwix-build/builds/49615228